lib/jnpr/junos/utils/sw.py

```diff
@@ -1007,8 +1010,11 @@ def poweroff(self, in_min=0):
 
         .. todo:: need to better handle the exception event.
         """
-        cmd = E('request-power-off', E('in', str(in_min)))
-
+        if hasattr(self._dev, '_is_linux') and self.dev.facts['_is_linux']:
```

Comment:
I don't understand this line. There is no `self._dev._is_linux` attribute, and there is no `self.dev` attribute.
I believe this line should be: `if self._dev.facts['_is_linux']:`

Reply:
@rsmekala Agee with @stacywsmith , why we need self._dev._is_linux?